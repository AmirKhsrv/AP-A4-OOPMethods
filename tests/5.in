best_book
show_sorted_shelf 47 currently_reading History
show_sorted_shelf 21 currently_reading HolidayHorror
best_reviewer
