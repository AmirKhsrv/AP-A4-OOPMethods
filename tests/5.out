id: 405
Title: Ellen Foster
Genre: Fiction
Author: Kaye Gibbons
Average Rating: 3.42
id: 163
Title: Comfort Object
Genre: Erotica
Author: Annabel Joseph
***
id: 404
Title: Monster Hunter Legion
Genre: Fantasy
Author: Larry Correia
***
id: 35
Title: No Strings Attached
Genre: Romance
Author: Nicolette Day
***
id: 269
Title: Seeds of Hate
Genre: Young Adult
Author: Melissa Perea
***
id: 319
Title: Smashed Squashed Splattered Chewed Chunked and Spewed
Genre: Fiction
Author: Lance Carbuncle
***
id: 198
Title: The Algebraist
Genre: Science Fiction
Author: Iain M. Banks
***
id: 286
Title: Three Daves
Genre: Romance
Author: Nicki Elson
***
id: 258
Title: Tiger's Voyage
Genre: Fantasy
Author: Colleen Houck
***
id: 233
Title: Beauty's Release
Genre: Adult Fiction
Author: A.N. Roquelaure
***
id: 14
Title: Descendant of the Crane
Genre: Fantasy
Author: Joan He
***
id: 119
Title: Digging Deep: Unearthing Your Creative Roots Through Gardening
Genre: Gardening
Author: Fran Sorin
***
id: 392
Title: From the Mixed-Up Files of Mrs. Basil E. Frankweiler
Genre: Fiction
Author: E.L. Konigsburg
***
id: 127
Title: Legend
Genre: Young Adult
Author: Marie Lu
***
id: 270
Title: Siddhartha
Genre: Classics
Author: Hermann Hesse
***
id: 54
Title: The 19th Wife
Genre: Historical
Author: David Ebershoff
***
id: 401
Title: The Age of Innocence
Genre: Classics
Author: Edith Wharton
***
id: 22
Title: The Children of Men
Genre: Fiction
Author: P.D. James
***
id: 286
Title: Three Daves
Genre: Romance
Author: Nicki Elson
***
id: 295
Title: Winnie The Pooh : The Complete Collection of Stories & Poems
Genre: Classics
Author: A.A. Milne
***
id: 31
Name: Athena Souri
Place of Birth: Switzerland
Member Since: December 24 2010
Favorite Genres: Romance
Favorite Authors: Carl Sagan, Colleen Clayton, Greer Noble
Number of Books in Read Shelf: 8
Number of Books in Want to Read Shelf: 15
Number of Books in Currently Reading Shelf: 6
Number of Likes: 2455
